app/controllers/webhooks_controller.rb

```diff
@@ -0,0 +1,27 @@
+class WebhooksController < ApplicationController
+  include ShopifyApp::WebhookVerification
+
+  class ShopifyApp::MissingWebhookJobError < StandardError; end
+
+  def receive
+    job_args = {shop_domain: shop_domain, webhook: webhook_params}
+    webhook_job_klass.perform_later(job_args)
+    head :ok
+  end
+
+  private
+
+  def webhook_params
+    params.except(:controller, :action, :type)
+  end
+
+  def webhook_job_klass
+    "#{webhook_type.classify}Job".safe_constantize
+  rescue NameError => e
```

Comment:
aaah woops this used to constantize. I should have looked into it before making that change suggested by @gmalette. This should be just constantize